File: bootstrap/v0.0.7.5/benchmarks/java/Factorial.java
public class Factorial {
    public static long factorial(int n) {
        if (n <= 1) {
            return 1;
        } else {
            return n * factorial(n - 1);
        }
    }

    public static void main(String[] args) {
        long sum = 0;
        int i = 1;
        while (i <= 20) {
            long result = factorial(i);
            sum = sum + result;
            i = i + 1;
        }
        System.exit(0);
    }
}

File: bootstrap/archived_runa_0.0.7.5+/v0.0.7.6/benchmarks/rust/factorial.rs
fn factorial(n: i64) -> i64 {
    if n <= 1 {
        1
    } else {
        n * factorial(n - 1)
    }
}

fn main() {
    let mut sum: i64 = 0;
    let mut i = 1;
    while i <= 20 {
        let result = factorial(i);
        sum = sum + result;
        i = i + 1;
    }
    std::process::exit(0);
}